api-docs-slate/source/includes/_wallet_tx.md

```diff
@@ -7,24 +7,35 @@ let id, hash
 ```
 
 ```shell--vars
-id="foo"
-hash="18d2cf5683d7befe06941f59b7fb4ca0e915dcb9c6aece4ce8966a29e7c576fe"
+id="primary"
+hash="0f2a1c65df0ea47c21c916071dd82228451a21c0a695e221bb9d61e96c6a89fd"
 ```
 
 ```shell--cli
 bwallet-cli --id=$id tx $hash
 ```
 
 ```shell--curl
-curl $url/wallet/$id/tx/$hash
+curl $walleturl/$id/tx/$hash
```

Comment:
don't these still have to have `/wallet/` in the path?

Reply:
I'm cheating a bit here: at the top of the docs I add a new environment variable:
in _wallet.md L#61:
```
# examples in these docs will use an environment variable:
walleturl=http://x:api-key@127.0.0.1:48334/wallet/
curl $walleturl/$id
```